pkg/controller/initializer.go

```diff
@@ -37,7 +37,12 @@ func init() {
 				log.Warn("OpenShift version could not be determined.")
 			}
 		}
-		kieapp.CreateConsoleYAMLSamples(&reconciler)
+		res, err := openshift.CompareOpenShiftVersion(nil, "4.3")
```

Comment:
what does this function do?

Reply:
this func makes sure the current platform is Openshift, and compare its version with the version passed in, then result the comparison result.
Here the code proceed to create the YAML samples only when this is OCP, and the version is >= 4.3.

Reply:
hmm, seems like this is duplicate code to what's being done above it. since the openshift version is already known at this point in the flow, couldn't we move this up into the `info.IsOpenShift()` if statement and do a simple `>= "4.3"`?

Reply:
modified.